README.md

```diff
@@ -97,39 +97,35 @@ runAlchemy();
 
 In addition to the built-in Ethers.js listeners, the Alchemy SDK includes support for Alchemy's Subscription API. This
 allows you to subscribe to events and receive updates
-as they occur. The two supported subscriptions are
-[alchemy_newFullPendingTransactions](https://alchemy.com/alchemy/enhanced-apis/subscription-api-websockets#alchemy_newfullpendingtransactions)
-and
-[alchemy_filteredNewFullPendingTransactions](https://alchemy.com/alchemy/enhanced-apis/subscription-api-websockets#alchemy_filterednewfullpendingtransactions)
-.
+as they occur. The main subscription API is
+[alchemy_pendingTransactions](https://docs.alchemy.com/alchemy/enhanced-apis/subscription-api-websockets#alchemy_pendingtransactions).
 
-The `Alchemy.getWebsocketProvider()` function configures the
-Alchemy [AlchemyWebSocketProvider](https://docs.ethers.io/v5/api/providers/api-providers/#AlchemyWebSocketProvider) and
-returns it. This can be used like the standard Ethers.js Websocket provider to add listeners for Alchemy events:
+The top level `Alchemy` instance can be used can be used like the standard Ethers.js [WebSocketProvider](https://docs.ethers.io/v5/api/providers/other/#WebSocketProvider) to add listeners for Alchemy events:
 
 ```ts
 import { Alchemy } from '@alch/alchemy-sdk';
 
 const alchemy = new Alchemy();
 
-const websocketProvider = alchemy.getWebsocketProvider();
-
 // Listen to all new pending transactions.
-websocketProvider.on(
+alchemy.on(
```

Comment:
This syntax is honestly a lot more confusing than web3.js (with "subscribe" being quite obvious). I'll have to update our docs well to make sure it's obvious what's happening

Reply:
yeah, ethers has a different model of doing subscriptions, and i tried to just add the additional overload here. lmk if you can think of another way to make it clearer!

Reply:
Yeah, we need to mirror Ethers if we want it to be a drop-in replacement, so it'll have to rely on documentation